docs/installation.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: default
+title: Installation Guide
+nav_order: 2
+---
+
+In order to use the MarkLogic Connector in an Anypoint project, you must add the connector to the project.
+This can be accomplished in two different ways. You can either use the built-in assistant, which will find the connector
+in Anypoint Exchange and then add it to your project. Or, if you have the connector in a Maven repository, you can add
+the dependency to the project and Anypoint will retrieve the connector from the repository.
+
+No matter which method is used, once the MarkLogic Connector has been successfully added to the Anypoint project, "MarkLogic"
+will appear as a module in the Mule Palette.
+
+## Installing from the Anypoint Exchange
+
+1. Open Anypoint Exchange by clicking on the "Open Exchange" near the top-left of the window.
```

Comment:
I think "icon" is needed after 'on the "Open Exchange"'. Given that the icons aren't intuitive either, I think a small screenshot may be needed here, unless you can think of words to describe its location (I was struggling to do this - e.g. "icon, which is typically found next to the 'New' icon in the Anypoint header.")